src/api/api.ts

```diff
@@ -0,0 +1,79 @@
+const BASE_URL = 'https://the-one-api.dev/v2';
+const ITEMS_ENDPOINT = '/character';
+const ITEMS_URL = BASE_URL + ITEMS_ENDPOINT;
+const TOKEN = 'qUlM6gAOQnY1o9NArsHP';
```

Comment:
Sensitive values like tokens should be passed through env variables (mostly because of security reasons). You can use `.env` files or use env variables directly.
https://12factor.net/config